src/main/java/tc/oc/pgm/rotation/Rotation.java

```diff
@@ -0,0 +1,130 @@
+package tc.oc.pgm.rotation;
+
+import com.google.common.base.Optional;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.bukkit.configuration.ConfigurationSection;
+import tc.oc.pgm.api.PGM;
+import tc.oc.pgm.map.PGMMap;
+
+public class Rotation implements PGMMapOrderProvider {
+  private ConfigurationSection configurationSection;
+
+  private String name;
+  private boolean enabled;
+  private List<PGMMap> maps = new ArrayList<>();
+  private int players;
+  private int position;
+
+  Rotation(ConfigurationSection configurationSection, String name) {
+    this.configurationSection = configurationSection;
+    this.name = name;
+    this.enabled = configurationSection.getBoolean("enabled");
+    this.players = configurationSection.getInt("players");
+    this.position = configurationSection.getInt("position");
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public void setName(String name) {
+    this.name = name;
+  }
+
+  public boolean isEnabled() {
+    return enabled;
+  }
+
+  public void setEnabled(boolean enabled) {
+    this.enabled = enabled;
+  }
+
+  public List<PGMMap> getMaps() {
+    return Collections.unmodifiableList(maps);
+  }
+
+  public void setMaps(List<PGMMap> maps) {
+    this.maps = maps;
+  }
+
+  public int getPlayers() {
+    return players;
+  }
+
+  public void setPlayers(int players) {
+    this.players = players;
+  }
+
+  public void setPosition(int position) {
+    this.position = position;
+  }
+
+  public int getPosition() {
+    return position;
+  }
+
+  public void load() {
+    configurationSection
+        .getStringList("maps")
+        .forEach(
+            map -> {
+              final Optional<PGMMap> mapByNameOrId =
+                  PGM.GLOBAL.get().getMapLibrary().getMapByNameOrId(map);
```

Comment:
Do not use `PGM.Global` instead do `PGM.get()`

Reply:
I mainly avoided `PGM.get()` because it figures as `@Depcrecated`. I guess I'll use it here anyways, but for future reference,  when should I use which?

Reply:
`PGM.get()`

Reply:
huh, I know I should use `PGM.get()` in this case,  but did you read my question? x.x

Reply:
You shouldn't use `GLOBAL` at all. This is probably just because it is an interface so the field is public. https://github.com/Electroid/PGM/blob/ae0ad4750e80ce6266da1d914fdc89e69ad61283/src/main/java/tc/oc/pgm/api/PGM.java#L61

Reply:
Okay, I will avoid using `GLOBAL` at all. Thanks for the clarification.

Reply:
Solved here: https://github.com/Electroid/PGM/pull/114/commits/223f8e1eaea620bb129a78e3b6c0eb629a5bc193